Tests.Unit.Core.ApplicationServices/ApplicationServices/UserServiceTest.cs

```diff
@@ -249,5 +281,21 @@ private void ExpectGetOrganizationReturns(Guid organizationId, Result<Organizati
         {
             _organizationServiceMock.Setup(x => x.GetOrganization(organizationId,OrganizationDataReadAccessLevel.All)).Returns(organization);
         }
+
+        private void ExpectTransactionBeginReturns()
+        {
+            var transaction = new Mock<IDatabaseTransaction>();
```

Comment:
let the helper return the mocked transaction so you can verify that it was committed